### ISSUE TITLE
Optimization, Thermalization, Equilibration, MDSim

### ISSUE THREAD

**Issue by kmb413** [kmb413/DecoyDiscrimination#13]
Can we do any of these 4 in a pure python way? The optimization part, maybe, if we do it using the scipy minimizer and make sure they're doing about the same thing, but what about the other steps?

**Comment by hainm** [comment]
No, you can't. And I dont thi k there is advantage to do so. I will reply with more detail when I am back from conference.
